addons/crm/models/crm_lead.py

```diff
@@ -353,6 +359,16 @@ def _compute_expected_revenue(self):
         for lead in self:
             lead.expected_revenue = round((lead.planned_revenue or 0.0) * (lead.probability or 0) / 100.0, 2)
 
+    @api.depends('recurring_revenue', 'recurring_plan')
+    def _compute_expected_monthly_revenue(self):
+        for lead in self:
+            lead.expected_monthly_revenue = round((lead.recurring_revenue or 0.0) / (lead.recurring_plan.number_of_months or 1), 2)
```

Comment:
Why do you need to use `round` ? (the `Monetary` field doesn't truncate the value based on the accuracy of the currency?)
(same question for `_compute_prorated_monthly_revenue`)